graphics/gimp2/Portfile

```diff
@@ -131,6 +132,13 @@ if {[vercmp $xcodeversion 4.3] < 0 && [string match "*macports*" ${configure.com
         RANLIB=${prefix}/bin/ranlib
 }
 
+# Temporary fix for linking against macOS SDK >10.13
+
+if {${os.major} > 13} {
+    configure.ldflags-append \
+                        -Wl,-sdk_version,10.9
```

Comment:
If this is only needed on darwin 14 and newer, do you know why the sdk is forced to 10.9 ? Presumably on say 10.13 it is using sdk 10.13 just fine, so does this still work correctly if you force this instead here ?

Reply:
Because if it does, would seem preferable to me to force the newest sdk that works.

Reply:
I chose to force 10.9 because the official builds are built using 10.9 SDK and the devs suggested me to do so. I'll try forcing 10.13 some time and report back.

Reply:
@cjones051073 Actually is there a way to know the version of the SDK that is being linked before configure process? Would be better to check if the actual SDK is bad (>10.13) than check if the OS version is >10.13.